docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md

```diff
@@ -8,6 +8,7 @@
 - [Step 3: Create a Cart](#step-3-create-a-cart)
```

Comment:
Remove the "Step #:" form all of these titles. All titles should also be in sentence case, per the style guide. See [Capitalization](https://intranet.bigcommerce.com/display/SST/Capitalization).